File: SledzSpecke.App/Models/Enums/SyncStatus.cs
namespace SledzSpecke.App.Models.Enums
{
    public enum SyncStatus
    {
        NotSynced,  // Nie zsynchronizowane z SMK
        Synced,     // Zsynchronizowane z SMK
        SyncFailed, // Próba synchronizacji nie powiodła się
        Modified,   // Zsynchronizowane, ale później zmodyfikowane
    }
}

File: SledzSpecke-Frontend/packages/shared/src/domain/value-objects/SyncStatus.ts
export enum SyncStatus {
  NotSynced = 'NotSynced',
  Synced = 'Synced',
  Modified = 'Modified',
  Approved = 'Approved'
}

export const isSyncStatusEditable = (status: SyncStatus): boolean => {
  return status !== SyncStatus.Approved;
};